patchview/README.patchview

```diff
@@ -0,0 +1,51 @@
+So what is patchview? It is a wrapper of filterdiff for use with numbered files.
+
+    $ patchview
+    (without args)
+    
+is equivalent to: lsdiff --number
+
+    $ patchview -F2-
+    (or with any other args)
+    
+is equivalent to: filterdiff -F2- (or whatever arguments are supplied)
+
+There are two scripts for working with git (gitdiff and gitdiffview) and two for svn (svndiff and svndiffview).
+
+    $ svndiff
+    $ gitdiff
+    (without args)
+    
+will give the list of files modified
+
+    $ svndiff -F1
+    $ gitdiff -F1
+    
+will show the patch of file #1
+
+    $ svndiffview
+    $ gitdiffview
+    
+pipe all patches through filterdiff to vim - -R (in read-only mode, easy to quit), showing complete patch with color.
+
+    $ svndiffview -F2
+    $ gitdiffview -F2
+    (or any other args)
+
+will pipe patch of file #2 to vim - -R
+
+and finally we can do scripts in one line, for example difftotrunk.sh script which compare 2 repos (trunk and devel)
+
+    diff ../trunk . -ru -x .svn | patchview $*
+
+to view differences of to directories and ./difftotrunk.sh (with args) will give the list of files that I change in devel
+
+    $ ./difftotrunk.sh , show all files and his number (that have differences).
+    $ ./difftotrunk.sh -F3,4 , will show the differences of files #3 and #4 only.
+    $ ./difftotrunk.sh -F3 -#1, will show only hunk #1 of file #3.
+    $ ./difftotrunk.sh -F3 -#x1, will show the differences of files #3 without hunk #1 (x means that exclude).
+
+
+In one second phase of project, gitdiff script was developed in Python, instead a very simple bash script, to allow git
```

Comment:
I don't really like the "second phase" part -- can we just describe what the current behaviour is?

Reply:
phase 1 is code , phase 2 is the scripts of patchview folder

Reply:
Now last commit is the phase 2 :smiley: